app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -37,8 +37,11 @@ def award_reputation_to_reviewers
         reviews = octokit_client.pull_request_reviews(repo, pr_id)
         reviewer_usernames = reviews.map { |reviewer| reviewer[:user][:login] }
 
-        reviewers = ::User.where(handle: reviewer_usernames)
+        reviewers = ::User.where(github_username: reviewer_usernames)
```

Comment:
This was an unfortunate mistake, which was not found due to me using the same handle and github_username in the test data 🤦